src/css/components/_captions-settings.scss

```diff
@@ -2,23 +2,32 @@
   background-color: $primary-background-color;
   background-color: rgba($primary-background-color, 0.75);
   color: $primary-foreground-color;
-  height: 70%;
+  height: 100%;
 }
 
 // Layout divs
 .vjs-text-track-settings .vjs-modal-dialog-content {
-  display: table;
+  display: grid;
```

Comment:
we can't just switch to CSS grid because it will break all older browsers that do not support it.

Reply:
We probably should keep what we have now and add a `@supports(grid)` (or whatever it is) for progressive enhancement. That way, older browsers will continue with what we have but newer browsers will get the grid.

Reply:
What are the browsers we need to support? CSS grid is supported in all modern browsers and partial support in ie10/11